demo/api/src/pages/pages.resolver.ts

```diff
@@ -67,12 +70,14 @@ export class PagesResolver {
             page.assign({
                 content: input.content.transformToBlockData(),
                 seo: input.seo.transformToBlockData(),
+                lastUpdatedUserLabel: `${user.name} (${user.email})`,
```

Comment:
hm, saving this as a string is not so nice, and having to manually set it for each and every mutation also isn't

maybe this can be implemented together with the user action log feature?

Reply:
Yes, I talke that over with @johnnyomair and we decided to put this feature on hold for now. We'll get back to it once user permissions and user action log work